index.html

```diff
@@ -1725,10 +1817,12 @@ <h5 id="Psionic Traits">Psionic Traits</h5>
             <tr>
                <td>Modify Memory
                </td>
-               <td>Modify any 10 minutes of memory from a creature from the last week. The
-                  creature must have an intelligence of 6 or more. The change in memory lasts
+               <td>
+                  You may modify any 10 minutes of memory from a creature from the last week. The
+                  creature must have an Intelligence of 6 or more. The change in memory lasts
```

Comment:
Is intelligence restriction needed?  They could modify the memory of a worm buttt it's not like it would do any good